### ISSUE TITLE
editrealtime is not working

### ISSUE THREAD

**Issue by alexandrecuer** [emoncms/emoncms#1821]
<img width="682" alt="image" src="https://user-images.githubusercontent.com/24553739/200304408-3e1b6d2c-1694-4029-9216-0d9b1f2cce7a.png">

**Comment by alexandrecuer** [comment]
php var `$vis_version` not declared in [Modules/vis/visualisations/editrealtime.php](https://github.com/emoncms/emoncms/blob/master/Modules/vis/visualisations/editrealtime.php)

dont know why we need give a v variable to feed.js and vis.helper.js ?
```
<script language="javascript" type="text/javascript" src="<?php echo $path;?>Modules/feed/feed.js?v=<?php echo $vis_version; ?>"></script>
<script language="javascript" type="text/javascript" src="<?php echo $path;?>Lib/vis.helper.js?v=<?php echo $vis_version; ?>"></script>
```

**Comment by borpin** [comment]
@TrystanLea bump 

https://community.openenergymonitor.org/t/datapoint-editing/22342/1

**Comment by TrystanLea** [comment]
Yes agreed, data editing needs some work. Id like to integrate this in the graph module ideally and remove the original editrealtime/daily interfaces.
